app/controllers/users_controller.rb

```diff
@@ -11,6 +11,13 @@ def update
     redirect_to root_path, notice: I18n.t(:user_data_updated)
   end
 
+  def unsubscribe
+    user_id = Rails.application.message_verifier(:unsubscribe).verify(params[:id])
+    User.find(user_id).unsubscribe_from_reminders!
```

Comment:
should I make this a class method in User and parametrize the message and id hash?